# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
# 
# Translators:
# Besnik Bleta <besnik@programeshqip.org>, 2024
# 
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2024-11-29 13:55+0100\n"
"PO-Revision-Date: 2017-08-09 10:34+0000\n"
"Last-Translator: Besnik Bleta <besnik@programeshqip.org>, 2024\n"
"Language-Team: Albanian (https://app.transifex.com/calamares/teams/20061/sq/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: sq\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: src/modules/bootloader/main.py:46
msgid "Install bootloader."
msgstr "Instalo ngarkues nisjesh."

#: src/modules/bootloader/main.py:671
msgid "Failed to install grub, no partitions defined in global storage"
msgstr ""
"S'u arrit të instalohej grub, te depozita globale s’ka të përkufizuara pjesë"

#: src/modules/bootloader/main.py:931
msgid "Bootloader installation error"
msgstr "Gabim instalimi Ngarkuesi Nisësi"

#: src/modules/bootloader/main.py:932
msgid ""
"The bootloader could not be installed. The installation command "
"<pre>{!s}</pre> returned error code {!s}."
msgstr ""
"Ngarkuesi i nisësit s’u instalua dot. Urdhri i instalimit <pre>{!s}</pre> u "
"përgjigj me kod gabimi {!s}."

#: src/modules/displaymanager/main.py:525
msgid "Cannot write LXDM configuration file"
msgstr "S’shkruhet dot kartelë formësimi LXDM"

#: src/modules/displaymanager/main.py:526
msgid "LXDM config file {!s} does not exist"
msgstr "S’ekziston kartelë formësimi LXDM {!s}"

#: src/modules/displaymanager/main.py:614
msgid "Cannot write LightDM configuration file"
msgstr "S’shkruhet dot kartelë formësimi LightDM"

#: src/modules/displaymanager/main.py:615
msgid "LightDM config file {!s} does not exist"
msgstr "S’ekziston kartelë formësimi LightDM {!s}"

#: src/modules/displaymanager/main.py:700
msgid "Cannot configure LightDM"
msgstr "S’formësohet dot LightDM"

#: src/modules/displaymanager/main.py:701
msgid "No LightDM greeter installed."
msgstr "S’ka të instaluar përshëndetës LightDM."

#: src/modules/displaymanager/main.py:732
msgid "Cannot write SLIM configuration file"
msgstr "S’shkruhet dot kartelë formësimi SLIM"

#: src/modules/displaymanager/main.py:733
msgid "SLIM config file {!s} does not exist"
msgstr "S’ekziston kartelë formësimi SLIM {!s}"

#: src/modules/displaymanager/main.py:956
msgid "No display managers selected for the displaymanager module."
msgstr "S’janë përzgjedhur përgjegjës ekrani për modulin displaymanager."

#: src/modules/displaymanager/main.py:957
msgid ""
"The displaymanagers list is empty or undefined in both globalstorage and "
"displaymanager.conf."
msgstr ""
"Lista “displaymanagers” është e zbrazët ose e papërkufizuar për të dy "
"rastet, për “globalstorage” dhe për “displaymanager.conf”."

#: src/modules/displaymanager/main.py:1044
msgid "Display manager configuration was incomplete"
msgstr "Formësimi i përgjegjësit të ekranit s’qe i plotë"

#: src/modules/dracut/main.py:29
msgid "Creating initramfs with dracut."
msgstr "Po krijohet initramfs me dracut."

#: src/modules/dracut/main.py:63
msgid "Failed to run dracut"
msgstr "S’u arrit të xhirohej dracut"

#: src/modules/dracut/main.py:64
#, python-brace-format
msgid "Dracut failed to run on the target with return code: {return_code}"
msgstr ""
"Dracut s’arriti të xhirojë me objektivin, kod përgjigjeje: {return_code}"

#: src/modules/dummypython/main.py:35
msgid "Dummy python job."
msgstr "Akt python dummy."

#: src/modules/dummypython/main.py:37 src/modules/dummypython/main.py:104
#: src/modules/dummypython/main.py:105
msgid "Dummy python step {}"
msgstr "Hap python {} dummy"

#: src/modules/fstab/main.py:29
msgid "Writing fstab."
msgstr "Po shkruhet fstab."

#: src/modules/fstab/main.py:382 src/modules/fstab/main.py:388
#: src/modules/fstab/main.py:416 src/modules/initcpiocfg/main.py:267
#: src/modules/initcpiocfg/main.py:271 src/modules/initramfscfg/main.py:85
#: src/modules/initramfscfg/main.py:89 src/modules/localecfg/main.py:140
#: src/modules/mount/main.py:344 src/modules/networkcfg/main.py:106
#: src/modules/openrcdmcryptcfg/main.py:72
#: src/modules/openrcdmcryptcfg/main.py:76 src/modules/rawfs/main.py:165
msgid "Configuration Error"
msgstr "Gabim Formësimi"

#: src/modules/fstab/main.py:383 src/modules/initramfscfg/main.py:86
#: src/modules/mount/main.py:345 src/modules/openrcdmcryptcfg/main.py:73
#: src/modules/rawfs/main.py:166
msgid "No partitions are defined for <pre>{!s}</pre> to use."
msgstr "S’ka pjesë të përkufizuara për <pre>{!s}</pre> për t’u përdorur."

#: src/modules/fstab/main.py:389 src/modules/initramfscfg/main.py:90
#: src/modules/localecfg/main.py:141 src/modules/networkcfg/main.py:107
#: src/modules/openrcdmcryptcfg/main.py:77
msgid "No root mount point is given for <pre>{!s}</pre> to use."
msgstr ""
"S’është dhënë pikë montimi rrënjë për <pre>{!s}</pre> për t’u përdorur."

#: src/modules/fstab/main.py:417
msgid "No <pre>{!s}</pre> configuration is given for <pre>{!s}</pre> to use."
msgstr ""
"S’është dhënë formësim <pre>{!s}</pre> për t’u përdorur nga <pre>{!s}</pre>."

#: src/modules/grubcfg/main.py:30
msgid "Configure GRUB."
msgstr "Formësoni GRUB-in."

#: src/modules/hwclock/main.py:26
msgid "Setting hardware clock."
msgstr "Po caktohet ora hardware."

#: src/modules/initcpiocfg/main.py:27
msgid "Configuring mkinitcpio."
msgstr "Po formësohet mkinitcpio."

#: src/modules/initcpiocfg/main.py:268
msgid "No partitions are defined for <pre>initcpiocfg</pre>."
msgstr "S’ka pjesë të përcaktuara për <pre>initcpiocfg</pre>."

#: src/modules/initcpiocfg/main.py:272
msgid "No root mount point for <pre>initcpiocfg</pre>."
msgstr "S’ka pikë montimi rrënjë për <pre>initcpiocfg</pre>."

#: src/modules/initramfscfg/main.py:32
msgid "Configuring initramfs."
msgstr "Po formësohet initramfs."

#: src/modules/localecfg/main.py:31
msgid "Configuring locales."
msgstr "Po formësohen vendoret."

#: src/modules/mkinitfs/main.py:27
msgid "Creating initramfs with mkinitfs."
msgstr "Po krijohet initramfs me mkinitfs."

#: src/modules/mkinitfs/main.py:49
msgid "Failed to run mkinitfs on the target"
msgstr "S’u arrit të xhirohej mkinitfs te objektivi"

#: src/modules/mkinitfs/main.py:50
msgid "The exit code was {}"
msgstr "Kodi i daljes qe {}"

#: src/modules/mount/main.py:43
msgid "Mounting partitions."
msgstr "Po montohen pjesë."

#: src/modules/mount/main.py:171 src/modules/mount/main.py:207
msgid "Internal error mounting zfs datasets"
msgstr "Gabim i brendshëm gjatë montimit të grupeve zfs të të dhënave"

#: src/modules/mount/main.py:183
msgid "Failed to import zpool"
msgstr "S’u arrit të importohej zpool"

#: src/modules/mount/main.py:199
msgid "Failed to unlock zpool"
msgstr "S’u arrit të shkyçej zpool"

#: src/modules/mount/main.py:216 src/modules/mount/main.py:221
msgid "Failed to set zfs mountpoint"
msgstr "S’u arrit të caktohej pikë montimi zfs"

#: src/modules/mount/main.py:386
msgid "zfs mounting error"
msgstr "Gabim montimi zfs"

#: src/modules/networkcfg/main.py:30
msgid "Saving network configuration."
msgstr "Po ruhet formësimi i rrjetit."

#: src/modules/openrcdmcryptcfg/main.py:26
msgid "Configuring OpenRC dmcrypt service."
msgstr "Po formësohet shërbim OpenRC dmcrypt."

#: src/modules/packages/main.py:54 src/modules/packages/main.py:65
#: src/modules/packages/main.py:75
msgid "Install packages."
msgstr "Instalo paketa."

#: src/modules/packages/main.py:63
#, python-format
msgid "Processing packages (%(count)d / %(total)d)"
msgstr "Po përpunohen paketat (%(count)d / %(total)d)"

#: src/modules/packages/main.py:68
#, python-format
msgid "Installing one package."
msgid_plural "Installing %(num)d packages."
msgstr[0] "Po instalohet një paketë."
msgstr[1] "Po instalohen %(num)d paketa."

#: src/modules/packages/main.py:71
#, python-format
msgid "Removing one package."
msgid_plural "Removing %(num)d packages."
msgstr[0] "Po hiqet një paketë."
msgstr[1] "Po hiqen %(num)d paketa."

#: src/modules/packages/main.py:775 src/modules/packages/main.py:787
#: src/modules/packages/main.py:815
msgid "Package Manager error"
msgstr "Gabim Përgjegjësi Paketash"

#: src/modules/packages/main.py:776
msgid ""
"The package manager could not prepare updates. The command <pre>{!s}</pre> "
"returned error code {!s}."
msgstr ""
"Përgjegjësi i paketave s’përgatiti dot përditësime. Urdhri <pre>{!s}</pre> u"
" përgjigj me kod gabimi {!s}."

#: src/modules/packages/main.py:788
msgid ""
"The package manager could not update the system. The command <pre>{!s}</pre>"
" returned error code {!s}."
msgstr ""
"Përgjegjësi i paketave s’përditësoi dot sistemin. Urdhri <pre>{!s}</pre> u "
"përgjigj me kod gabimi {!s}."

#: src/modules/packages/main.py:816
msgid ""
"The package manager could not make changes to the installed system. The "
"command <pre>{!s}</pre> returned error code {!s}."
msgstr ""
"Përgjegjësi i paketave s’bëri dot ndryshime te sistemi i instaluar. Urdhri "
"<pre>{!s}</pre> u përgjigj me kod gabimi {!s}."

#: src/modules/plymouthcfg/main.py:27
msgid "Configure Plymouth theme"
msgstr "Formësoni temën Plimuth"

#: src/modules/rawfs/main.py:26
msgid "Installing data."
msgstr "Po instalohen të dhëna."

#: src/modules/services-openrc/main.py:29
msgid "Configure OpenRC services"
msgstr "Formësoni shërbime OpenRC"

#: src/modules/services-openrc/main.py:57
msgid "Cannot add service {name!s} to run-level {level!s}."
msgstr "S’shtohet dot shërbimi {name!s} te run-level {level!s}."

#: src/modules/services-openrc/main.py:59
msgid "Cannot remove service {name!s} from run-level {level!s}."
msgstr "S’hiqet dot shërbimi {name!s} nga run-level {level!s}."

#: src/modules/services-openrc/main.py:61
msgid ""
"Unknown service-action <code>{arg!s}</code> for service {name!s} in run-"
"level {level!s}."
msgstr ""
"Service-action <code>{arg!s}</code> i panjohur për shërbimin {name!s} te "
"run-level {level!s}."

#: src/modules/services-openrc/main.py:93
msgid "Cannot modify service"
msgstr "S’modifikohet dot shërbimi"

#: src/modules/services-openrc/main.py:94
msgid ""
"<code>rc-update {arg!s}</code> call in chroot returned error code {num!s}."
msgstr ""
"Thirrje <code>rc-update {arg!s}</code> në chroot u përgjigj me kod gabimi "
"{num!s}."

#: src/modules/services-openrc/main.py:101
msgid "Target runlevel does not exist"
msgstr "Runlevel-i i synuar nuk ekziston"

#: src/modules/services-openrc/main.py:102
msgid ""
"The path for runlevel {level!s} is <code>{path!s}</code>, which does not "
"exist."
msgstr ""
"Shtegu për runlevel {level!s} është <code>{path!s}</code>, i cili nuk "
"ekziston."

#: src/modules/services-openrc/main.py:110
msgid "Target service does not exist"
msgstr "Shërbimi i synuar nuk ekziston"

#: src/modules/services-openrc/main.py:111
msgid ""
"The path for service {name!s} is <code>{path!s}</code>, which does not "
"exist."
msgstr ""
"Shtegu për shërbimin {name!s} është <code>{path!s}</code>, i cili nuk "
"ekziston."

#: src/modules/services-systemd/main.py:26
msgid "Configure systemd units"
msgstr "Formësoni njësi systemd-je"

#: src/modules/services-systemd/main.py:64
msgid "Cannot modify unit"
msgstr "S’ndryshohet dot njësi"

#: src/modules/services-systemd/main.py:65
msgid ""
"<code>systemctl {_action!s}</code> call in chroot returned error code "
"{_exit_code!s}."
msgstr ""
"Thirrja <code>systemctl {_action!s}</code> në chroot mori si përgjigje kodin"
" e gabimit {_exit_code!s}."

#: src/modules/services-systemd/main.py:66
msgid "Cannot {_action!s} systemd unit <code>{_name!s}</code>."
msgstr ""

#: src/modules/unpackfs/main.py:34
msgid "Filling up filesystems."
msgstr "Po mbushen sisteme kartelash."

#: src/modules/unpackfs/main.py:255
msgid "rsync failed with error code {}."
msgstr "rsync dështoi me kod gabimi {}."

#: src/modules/unpackfs/main.py:300
msgid "Unpacking image {}/{}, file {}/{}"
msgstr "Po shpaketohet paketa {}/{}, kartela {}/{}"

#: src/modules/unpackfs/main.py:315
msgid "Starting to unpack {}"
msgstr "Po fillohet të shpaketohet {}"

#: src/modules/unpackfs/main.py:324 src/modules/unpackfs/main.py:480
msgid "Failed to unpack image \"{}\""
msgstr "Dështoi shpaketimi i figurës “{}”"

#: src/modules/unpackfs/main.py:443
msgid "No mount point for root partition"
msgstr "S’ka pikë montimi për ndarjen rrënjë"

#: src/modules/unpackfs/main.py:444
msgid "globalstorage does not contain a \"rootMountPoint\" key."
msgstr "globalstorage nuk përmban kyç “rootMountPoint”."

#: src/modules/unpackfs/main.py:447
msgid "Bad mount point for root partition"
msgstr "Pikë e gabuar montimi për ndarjen rrënjë"

#: src/modules/unpackfs/main.py:448
msgid "rootMountPoint is \"{}\", which does not exist."
msgstr "rootMountPoint është “{}”, çka nuk ekziston."

#: src/modules/unpackfs/main.py:452 src/modules/unpackfs/main.py:468
#: src/modules/unpackfs/main.py:472 src/modules/unpackfs/main.py:478
#: src/modules/unpackfs/main.py:515
msgid "Bad unpackfs configuration"
msgstr "Formësim gabim i unpackfs-ë"

#: src/modules/unpackfs/main.py:453
msgid "There is no configuration information."
msgstr "S’ka hollësi formësimi."

#: src/modules/unpackfs/main.py:469
msgid "The filesystem for \"{}\" ({}) is not supported by your current kernel"
msgstr ""
"Sistemi i kartelave për \"{}\" ({}) nuk mbulohet nga kerneli juaj i tanishëm"

#: src/modules/unpackfs/main.py:473
msgid "The source filesystem \"{}\" does not exist"
msgstr "Sistemi i kartelave “{}” ({}) s’ekziston"

#: src/modules/unpackfs/main.py:479
msgid ""
"Failed to find unsquashfs, make sure you have the squashfs-tools package "
"installed."
msgstr ""
"S’u arrit të gjendet unsquashfs, sigurohuni se keni të instaluar paketën "
"squashfs-tools."

#: src/modules/unpackfs/main.py:516
msgid "The destination \"{}\" in the target system is not a directory"
msgstr "Vendmbërritja “{}” te sistemi i synuar s’është drejtori"

#: src/modules/zfshostid/main.py:27
msgid "Copying zfs generated hostid."
msgstr "Po kopjohet hostid prodhuar nga zfs."
